packages/create-react-native-library/templates/nitro-module/$nitro.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
What's up with the `$` prefix in the filename?

Reply:
removed them 👍